plugins/@grouparoo/pardot/src/lib/connect.ts

```diff
@@ -0,0 +1,19 @@
+import { SimpleAppOptions } from "@grouparoo/core";
+import PardotClient from "./client";
+
+export async function connect(appOptions: SimpleAppOptions) {
+  const client = new PardotClient({
+    businessUnitId: appOptions.businessUnitId,
+    // you can change loginUrl and pardotUrl to connect to sandbox or prerelease env.
+    // loginUrl: "https://test.salesforce.com"
+    pardotUrl: "https://pi.demo.pardot.com",
```

Comment:
I wonder if we need a checkbox or something in the options for "demo" or "sandbox environment" or something. Some people might be using that when testing things out.

Reply:
Thought about this too, but just went with what was being done in salesforce. It's interesting here because there can be multiple configs where you may use any combination of test.salesforce and pi.demo.pardot

See table in https://developer.pardot.com/kb/authentication/

![image](https://user-images.githubusercontent.com/4368928/109699713-45555400-7b67-11eb-8445-574c6c146eae.png)

If there's a way to do checkboxes in the config, we could do a checkbox for each (demo pardot, test salesforce) or just allow changing the url itself.

Reply:
gross.
i guess let's do 2 optional dropdowns and default to "Production Account" if not filled out in each case.

salesforce_domain: [login.salesforce.com, test.salesforce.com] (default to login)
pardot_domain: [pi.pardot.com, pi.demo.pardot.com] (default to non-demo)